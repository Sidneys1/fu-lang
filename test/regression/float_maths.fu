main: i32() = {
  a: f32 = 50f;
  b: f32 = 4f;
  c: f32 = a - b; // 46f
  d: f32 = a * b; // 200f
  e: f32 = a / b; // 12.5f
  return c + d + e; // 46 + 200 + 12.5 = 258.5 (258 as int)
};
