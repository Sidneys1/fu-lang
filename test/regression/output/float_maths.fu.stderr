x x x x x x Failed to lex `Identity`: Expected Colon
INFO:fu.compiler:Output binary `a.out`: 554 Bytes
